packages/api/api-generator/src/commands/generate/utils/__tests__/remove-unused-imports.spec.ts

```diff
@@ -0,0 +1,102 @@
+import ts from "typescript";
+
+import { removeUnusedImports } from "../write-generated-file";
+
+function runTransform(input: string): string {
+    input = input.trim().replace(/^\s+/gm, "");
+    const sourceFile = ts.createSourceFile("file.ts", input, ts.ScriptTarget.Latest, true, ts.ScriptKind.TS);
```

Comment:
```suggestion
    const sourceFile = ts.createSourceFile("file.ts", input, ts.ScriptTarget.ES2024, true, ts.ScriptKind.TS);
```

Probably need to change it in the test as well...